daemons/fenced/fenced_history.c

```diff
@@ -224,12 +329,14 @@ stonith_merge_in_history_list(GHashTable *history)
     GHashTableIter iter;
     remote_fencing_op_t *op = NULL;
     gboolean updated = FALSE;
+    guint op_list_size;
```

Comment:
Not needed

Reply:
left over from checking the length outside of the trim-call.
actually interesting that the compiler didn't moan.